js/backgroundExecutor.js

```diff
@@ -0,0 +1,62 @@
+import {getActiveTenant} from "/js/tenants.js";
```

Comment:
This file is just `js/background.js` renamed, to save probably just my confusion with having `background.js` in the root as well now.